charts/timber-fluentd/templates/hpa.yaml

```diff
@@ -0,0 +1,23 @@
+{{- if .Values.autoscaling.enabled }}
+apiVersion: autoscaling/v1
+kind: HorizontalPodAutoscaler
+metadata:
+  name: {{ include "timber-fluentd.fullname" . }}-fluentd
```

Comment:
We should remove `-fluentd` since `{{ include "timber-fluentd.fullname" . }}` generate string that ends with `fluentd` and hence the resource name become repeated (e.g. `project-model-timber-fluentd-fluentd`